lib/drives/pages/drive_detail_page.dart

```diff
@@ -155,36 +155,27 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
             return Padding(
               padding: const EdgeInsets.only(left: 8.0),
               child: Column(
-                children: <Widget>[
-                  const SizedBox(height: 10.0),
-                  Container(
-                    decoration: BoxDecoration(
-                      border: Border.all(color: Theme.of(context).colorScheme.onSurface.withOpacity(0.5)),
-                      borderRadius: BorderRadius.circular(5.0),
-                    ),
-                    padding: const EdgeInsets.all(8.0),
-                    width: double.infinity,
-                    child: Column(
-                      key: const Key('waypointCard'),
-                      mainAxisSize: MainAxisSize.min,
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: buildCard(stop),
-                    ),
-                  ),
-                  const SizedBox(height: 10.0)
-                ],
+                children: <Widget>[const SizedBox(height: 5), buildCard(stop), const SizedBox(height: 5)],
```

Comment:
Padding?

Reply:
This is actually a good example where I think I accomplished the goal of making the app feel less cluttered without increasing the padding. With the cards instead of containers with borders, it feels (at least to me) like there is just one less element on the page, and I think this looks quite balanced overall now.

Reply:
No I mean why did you use SizedBoxes in a Column instead of Padding?